mindsdb/utilities/log.py

```diff
@@ -39,24 +115,15 @@ def initialize_log(config, logger_name='main', wrap_print=False):
     ch = logging.StreamHandler()
     ch.setLevel(config['log']['level']['console'])       # that level will be in console
     log.addHandler(ch)
+    db_handler = DbHandler()
+    log.addHandler(db_handler)
 
     log_path = os.path.join(config.paths['log'], logger_name)
     if not os.path.isdir(log_path):
         os.mkdir(log_path)
 
-    fh = logging.handlers.RotatingFileHandler(
-        os.path.join(log_path, 'log.txt'),
-        mode='a',
-        encoding='utf-8',
-        maxBytes=100 * 1024,
-        backupCount=3
-    )
-    fh.setLevel(config['log']['level']['file'])
-    fh.setFormatter(formatter)
```

Comment:
`formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')`
formatter is not used now, probably can be removed

Reply:
(above)